src/main/java/dev/jbang/cli/Info.java

```diff
@@ -304,7 +362,68 @@ class Jar extends BaseInfoCommand {
 	@Override
 	public Integer doCall() throws IOException {
 		ScriptInfo info = getInfo(false);
-		System.out.println(info.applicationJar);
+		out.println(info.applicationJar);
 		return EXIT_OK;
 	}
 }
+
+@CommandLine.Command(name = "docs", description = "Open the documentation file in the default browser.")
+class Docs extends BaseInfoCommand {
+
+	@Option(names = {
+			"--open" }, negatable = true, defaultValue = "false", description = "Open the (first) documentation file/link in the default browser")
+	public boolean open;
+
+	@Parameters(arity = "0..*", description = "Documentation file/links to open")
+	List<String> docsToOpen;
+
+	@Override
+	public Integer doCall() throws IOException {
+
+		ScriptInfo info = getInfo(false);
+
+		URI toOpen[] = new URI[1];
```

Comment:
There is a practical reason for this instead of:
```suggestion
		URI[] toOpen = new URI[1];
```

?

Reply:
Do we really need an array of 1 URI instead of of just an URI which could be empty? In case we could used `Optional<URI>`, which has a plus on readability of the intention, but has a minus on object allocation, however not impactful because just one allocation per iteration and we aren't looping on big collection.